Testscripts/Windows/VERIFY-VF-HOT-REMOVE-XDP.ps1

```diff
@@ -0,0 +1,171 @@
+        }
+        $installXDPCommand = @"
+./XDPDumpSetup.sh $($receiverVMData.InternalIP) $iFaceName 2>&1 > ~/xdpConsoleLogs.txt
+. utils.sh
+collect_VM_properties
+"@
+        Set-Content "$LogDir\StartXDPSetup.sh" $installXDPCommand
+        Copy-RemoteFiles -uploadTo $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+            -files "$LogDir\StartXDPSetup.sh" `
+            -username $user -password $password -upload -runAsSudo
+
+        Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+            -username $user -password $password -command "chmod +x *.sh" -runAsSudo | Out-Null
+        $testJob = Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+            -username $user -password $password -command "./StartXDPSetup.sh" `
+            -RunInBackground -runAsSudo -ignoreLinuxExitCode
+        $timer = 0
+        while ($testJob -and ((Get-Job -Id $testJob).State -eq "Running")) {
+            $currentStatus = Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+                -username $user -password $password -command "tail -2 ~/xdpConsoleLogs.txt | head -1" -runAsSudo
+            Write-LogInfo "Current Test Status: $currentStatus"
+            Wait-Time -seconds 20
+            $timer += 1
+            if ($timer -gt 15) {
+                Throw "XDPSetup did not stop after 5 mins. Please check logs."
+            }
+        }
+
+        $currentState = Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+            -username $user -password $password -command "cat state.txt" -runAsSudo
+        if ($currentState -imatch "TestCompleted") {
+            Write-LogInfo "XDPSetup successfully ran on $($receiverVMDAta.RoleName)"
+            # Start ping and xdpdump
+            $ping_command = "ping -I $iFaceName -c 300 $($senderVMData.SecondInternalIP) > ~/pingOut$LogSuffix.txt"
+            Write-LogInfo "Starting command: $ping_command on $($receiverVMData.RoleName)"
+            Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort -username $user -password $password `
+                -command $ping_command -RunInBackground -runAsSudo
+            Write-LogDbg "XDP program cannot run with LRO (RSC) enabled, disable LRO before running XDP"
+            $xdp_command = "ethtool -K $iFaceName lro off && cd ~/bpf-samples/xdpdump && ./xdpdump -i $iFaceName > ~/xdpdumpout_VFTest.txt 2>&1"
+            $testJob = Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort -username $user -password $password `
+                -command $xdp_command -RunInBackground -runAsSudo -ignoreLinuxExitCode
+
+            Wait-Time -seconds 10
+            Set_VF $receiverVMData $false
+            Wait-Time -seconds 20
+            Set_VF $receiverVMData $true
+            Wait-Time -seconds 10
+            # Kill and verify XDP unloaded successfully

```

Comment:
Conclude the test results based on the counters of rx captured at different stages.